fluentd.gemspec

```diff
@@ -65,6 +65,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency("test-unit", ["~> 3.3"])
   gem.add_development_dependency("test-unit-rr", ["~> 1.0"])
   gem.add_development_dependency("oj", [">= 2.14", "< 4"])
+  gem.add_development_dependency("async", ">= 2.0")
```

Comment:
Does this need to be `development_dependency`?

Reply:
If not specified, async v1.x would also be available. However, this does not work in practice, as the corresponding code is erased.

Or, it increases the version of async-http. 
It would  increase the required version of async.
https://github.com/fluent/fluentd/blob/c2b09d4396c0f065f60099d3e84ca52ec5faf4f8/fluentd.gemspec#L68

https://rubygems.org/gems/async-http/versions/0.50.0

Reply:
I adjust the version of `async-http` instead.

Reply:
Thanks!
Note:

* `async` is not mandatory for Fluentd. If there is no `async`, `webrick` is used for `HttpServer` plugin helper.
  * This should be fixed in another PR in the future....
* Fluentd uses `async` and `async-http`, and `async-http` depends on `async`. So it is enough to manage `async-http` version.